DataAccess/Model/Word.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccess.Model
+{
+    public class Word
+    {
+        /// <summary>
+        /// The letters of a word in their correct order.
+        /// </summary>
+        public string Letters { get; set; }
```

Comment:
hier können wir evtl nen Char array draus machen da es eine string.ToCharArray Methode gibt die einen Char Array erstellt und wir so das ganze nur einmal machen müssen und die Funktion nicht immer aufrufen müssen

Reply:
Since we can call String[Index] this isn't neccessary. The council has deemed your proposal superfluous.